addons/editor-compact/userstyle.css

```diff
@@ -0,0 +1,422 @@
+/* Full page */
```

Comment:
the tutorials and logs popup are not compact (they should have the same classes?)
![image](https://user-images.githubusercontent.com/72760579/195226230-e1b94020-df14-4db1-bdc0-543ed41d512b.png)